4_generator.go

```diff
@@ -60,6 +63,14 @@ func GenerateFromContext(g *client.GitClient, contexts []*TemplateContext, forma
 				"unescape": func(s string) template.HTML {
 					return template.HTML(s)
 				},
+				"gsub": func(pattern string, repl string, s string) string {
```

Comment:
What is the full name of `gsub`? I think the name might not be accurate.

Reply:
gsub, hm... yes: I related to "sub" "gsub" : https://www.journaldev.com/43690/sub-and-gsub-function-r

What is your suggestion: `regex_repl` or something shorter?

Reply:
I think `replace` is good for me. even it replaced with regular expression

Reply:
Ok. See the last 2 Reviews.
Instead of baking our own template functions I added the sprig library. Overwrote the regex functions wich cached ones for better performance.
In that way we can really use the template engine quite good =). And no more PRs because stuff is missing. Most thing you ever need are there.

Hope you are fine with that?

Reply:
I also removed the paddingLeft function as IMO its not good bahavior to preindent text.
Leave it as is an leave that to the user. OK that might break in user perspective, but that case is rare, as they could not have used much template function on it, and were basically limited to have it used in the same fashion as the default template.